### DISCUSSION TITLE
htaccess auth type styl

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#477] [General]
Was this ever implemented as a login method? I thought I read it in a previous build?

**Comment by rejetto** [comment]
yes, it should work correctly in 0.52

**Reply by Stoufs** [reply]
I am not seeing that option anywhere?

**Comment by rejetto** [comment]
it's not an option, it's just supported

**Reply by Stoufs** [reply]
Is there a guide to configuring it?

**Comment by rejetto** [comment]
say clearly what you are trying to achieve

**Reply by Stoufs** [reply]
Apologies. I am trying to setup authentication so that httaccess type style login prompt is used.

**Reply by Stoufs** [reply]
Like this:

https://help.one.com/hc/article_attachments/360008648857

**Comment by rejetto** [comment]
thanks for the clarification.
That mode is not used by the frontend.
At the moment it is triggered only when trying to access a protected file (not a folder).
Why would you want that?

**Reply by Stoufs** [reply]
I see what youre saying now. I tested this and it does bring up that style to a protected file. My only concern is that since this is a differnent type of login style then the default one, when it returns an unauthorized message, it doesn't show the same custom un auth error I setup in custom html linked to your default login prompt. Any way to fix this?

**Comment by rejetto** [comment]
so, you'd want to see your custom HTML when failing the "htaccess" login?

**Reply by Stoufs** [reply]
Correct!

**Comment by rejetto** [comment]
sounds fair, will be ready for next release

**Reply by Stoufs** [reply]
Any eta on next release?

**Reply by rejetto** [reply]
less than a week, i hope
